WalletWasabi/JsonConverters/KeyJsonConverter.cs

```diff
@@ -17,7 +17,18 @@ public override bool CanConvert(Type objectType)
 		/// <inheritdoc />
 		public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
 		{
-			return Key.Parse(((string)reader.Value).Trim());
+			var serializedKey = ((string)reader.Value).Trim();
+			foreach (var network in Network.GetNetworks())
```

Comment:
Can we order the Networks to get the Main first, to reduce the number of exceptions? Or is it ordered already?

Reply:
https://github.com/MetacoSA/NBitcoin/blob/master/NBitcoin/Network.cs#L2680